src/jit/gentree.cpp

```diff
@@ -394,6 +408,88 @@ bool GenTree::IsNodeProperlySized() const
 }
 #endif
 
+/*****************************************************************************
+ *
+ *  When 'NODEBASH_STATS' is enabled in "jit.h" we record all instances of
+ *  an existing GenTree node having its operator changed. This can be useful
+ *  for two (related) things - to see what is being bashed (and what isn't),
+ *  and to verify that the existing choices for what nodes are marked 'large'
+ *  are reasonable (to minimize "wasted" space).
+ *
+ *  And yes, the hash function / logic is simplistic, but it is conflict-free
```

Comment:
Oh, BTW, memory alloc was indeed the main issue I ran into when I tried to use one of the existing hash classes; since this code is strictly for an occasional quick check by a developer and should never be enabled in any "real" build, I simply took this shortcut. Of course that same argument also says that we might as well use whatever you guys think is cleanest (since it won't matter for real use anyway), so what do you recommend?